docker-local/wallet-cdc/application.properties

```diff
@@ -0,0 +1,46 @@
+application.name=@project.artifactId@
```

Comment:
is this file needed? i see that the only diff here is the mongo connection string with the src/resources one for the replicaSet option.

Can we modify the src/resources one to add an optional parameter for the replicaset part of the connection string (as done [here](https://github.com/pagopa/pagopa-ecommerce-helpdesk-service/blob/890ec2563f4cfc9484e39b884d71008af8e91805/src/main/resources/application.properties#L26) for example)?

this way we can avoid duplicate this file

Reply:
unfortunately the real problem is authentication, so username and password that enabling replicaset require a somewhat complex configuration on the mongo container, so this was a workaround to avoid changing the connection string and maybe taking the whole string in an environment variable (even if then more difficult to do tuning on the parameters). to  refactor on that part would have intervened too much on the application part.

Reply:
can be reviewed in a dedicated PR

Reply:
perfect!